smqtk_iqr/web/iqr_service/iqr_server.py

```diff
@@ -188,6 +196,20 @@ def __init__(self, json_config: Dict):
 
         # Initialize from config
         self.positive_seed_neighbors = sc_config['positive_seed_neighbors']
+        self.autoneg_select_ratio = sc_config['autoneg_select_ratio']
+
+        metric_map: Dict[str, Callable[[np.ndarray, np.ndarray], np.ndarray]] = {
+            "euclidean": euclidean_distance,
+            "cosine": cosine_distance,
+            "histogram_intersection": histogram_intersection_distance,
+        }
+        try:
+            self.distance_metric = metric_map[sc_config['distance_metric']]
+        except KeyError:
+            LOG.warning("Invalid distance metric '{}'. "
```

Comment:
This is an error, so we should log using the same severity
```suggestion
            LOG.error("Invalid distance metric '{}'. "
```